mantidproject/mantid
# Muon check field type

**Description of work.**

This adds checks to the new muon GUI that the loaded run set is all for the same main field direction and warns the user if not.

It also adds the grouping files for CHRONUS and updates the CHRONUS IDF to be correct.

**Report to:** Anthony 

**To test:**

For MUSR 62250 is a longitudinal run and 62260 is a transverse run. Try loading these in various combinations and check you are warned appropriately.

Try loading some CHRONUS data and check that a default grouping table is created. 5000 and 8000 are runs numbers with different fields.



Fixes #24889  


*This does not require release notes* because the new GUI is not yet user facing

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
